### ISSUE TITLE
why can't I reproduce the mAP provided by README.md on COCO dataset by training from scratch?

### ISSUE THREAD

**Issue by xiaochengsky** [ultralytics/yolov5#5289]
## ❔Question
1) I used the 'get_coco.sh' to download the COCO datasets and used master branch(v6.0) to training;
2) I used the method recommended by README.md to reproduce the mAP, but I failed. For instance, I used 'python train.py --data coco.yaml --cfg yolov5s.yaml --weights ' ' --batch-size 64 --device 0' , then I got the best mAP is 36.2%(the best mAP verified at the end of each epoch, 5000 images), it still has a gap of 1 mAP(37.2%). Also, My GPU has about 16GB(15.1G) memory, and I think it may be enough. 

So, I have some question, and hope to get answers:
1) I found a information, when the project load the COCO dataset, it shows 1021 images is missing in train dataset, and 48 images is missing in val dataset, I don't know if this is main reason why I can't reproduce the mAP.
2) Also, I want to know the COCO dataset which is downloaded by 'get_coco.sh', has any difference with original COCO dataset from the Microsoft in using the labels, except for bbox.  such as 'iscrowd'?

**Comment by glenn-jocher** [comment]
@xiaochengsky smaller models (n, s) should be trained with low augmentation, and larger models (m, l, x) should use hyp.scratch-high.yaml for best results.
```
python train.py --data coco.yaml --cfg yolov5s.yaml --weights ' ' --batch-size 64 --device 0 --hyp hyp.scratch-low.yaml
```

**Comment by xiaochengsky** [comment]
@glenn-jocher 
Wow, thanks for your reply, and I will try it right away.

**Comment by xiaochengsky** [comment]
@glenn-jocher 
Hi, according to the advice you gave, I still can't reproduce the mAP. 
I need to reproduce these mAP, because I want to do some experiences in yolov5. Can you help me?
Some logs during my training are as follows:
1) when loading data, I found some data is missing:
===========================================================================================
Scaled weight_decay = 0.0005                                                                                                                                                                                                                                                      
optimizer: SGD with parameter groups 57 weight, 60 weight (no decay), 60 bias                                                                                                                                                                                                     
train: Scanning '../datasets/coco/train2017.cache' images and labels... 117266 found, 1021 missing, 0 empty, 0 corrupted: 100%|████████████████████████████████████████████| 118287/118287 [00:00<?, ?it/s]                                                                       
val: Scanning '../datasets/coco/val2017.cache' images and labels... 4952 found, 48 missing, 0 empty, 0 corrupted: 100%|████████████████████████████████████████████████████████| 5000/5000 [00:00<?, ?it/s]                                                                       
Plotting labels...                                                                                                                                                                                                                                                                
===========================================================================================

2) When the  training is completed, the mAP is still low(35.9%):
===========================================================================================
     Epoch   gpu_mem       box       obj       cls    labels  img_size                
   299/299       12G   0.04242   0.06151    0.0152       220       640: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1849/1849 [24:47<00:00,  1.24it/s]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 40/40 [00:33<00:00,  1.19it/s]
                 all       5000      36335      0.632      0.532      0.558      0.358
                                                                                      
300 epochs completed in 129.039 hours.                                                
Optimizer stripped from runs/train/exp5/weights/last.pt, 14.8MB                       
Optimizer stripped from runs/train/exp5/weights/best.pt, 14.8MB                       
                                                                                      
Validating runs/train/exp5/weights/best.pt...                                         
Fusing layers...                                                                      
Model Summary: 213 layers, 7225885 parameters, 0 gradients, 16.5 GFLOPs               
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 40/40 [01:01<00:00,  1.53s/it]
                 all       5000      36335      0.674      0.511       0.56      0.359
===========================================================================================

I'm looking forward for your reply, THX!

**Comment by glenn-jocher** [comment]
@xiaochengsky reported mAP is pycocotools mAP

**Comment by xador70** [comment]
I'm still facing the same issue with downloading the COCO dataset with "bash data/scripts/get_coco.sh".

In the training data, 117266 found, 1021 missing
and in the validation data, 4952 found, 48 missing, 0 empty

I would appreciate it if you can help me. Thanks!

**Comment by SikaAntler** [comment]
Hello, did you solve the problem? I'm also meet this. I was trying to reproduce the 64.1 mAP(0.5) by yolov5m on COCO dataset. My training shell script is:
`python -m torch.distributed.run --nproc_per_node 4 train.py --weights '' --cfg yolov5m.yaml --data data/coco.yaml --hyp data/hyps/hyp.scratch-high.yaml --epochs 300 --batch-size 128 --imgsz 640 --device 0,1,2,3 --sync-bn`
However, I could only get the best mAP(0.5) of **0.58691**, I'm very confused about this.
